geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ConverterIngest.scala

```diff
@@ -104,8 +98,11 @@ class ConverterIngest(dsParams: Map[String, String],
               toWrite.getUserData.put(Hints.USE_PROVIDED_FID, java.lang.Boolean.TRUE)
               try {
                 fw.write()
+                written.incrementAndGet()
```

Comment:
Unfortunately I don't know that the FW in it's current implementation super supports that. It would be nice to have a transaction that did that for us...exposed the flush time, flush ack, callback something in the API